src/commands/Info/commandstat.js

```diff
@@ -0,0 +1,18 @@
+exports.run = async function (bot, msg, args) {
```

Comment:
We usually use lambdas, not functions. For the sake of consistency, it would probably be better to change this to `async (bot, msg, args) => {`.